src/test/scala/io/github/spark_redshift_community/spark/redshift/ConversionsSuite.scala

```diff
@@ -58,6 +58,20 @@ class ConversionsSuite extends FunSuite {
     assert(convertedRow == expectedRow)
   }
 
+  test("Regression test for parsing timestamptz (bug #25 in spark_redshift_community)") {
+    val rowConverter = createRowConverter(
+      StructType(Seq(StructField("timestampWithTimezone", TimestampType))))
+
+    val timestampWithTimezone = "2014-03-01 00:00:01.123 -0300"
+    val expectedTimestampWithTimezoneMillis = TestUtils.toMillis(
+      2014, 2, 1, 0, 0, 1, 123)
+
+    val convertedRow = rowConverter(Array(timestampWithTimezone))
+    val expectedRow = Row(new Timestamp(expectedTimestampWithTimezoneMillis))
+
+    assert(convertedRow == expectedRow)
+  }
```

Comment:
Can we further add a test case to cover the Catch clause that handles the input without timezone data?

Reply:
We already do have that test for all normal timestamps (we've always had it). Please check line 45 in the ConversionSuite: https://github.com/spark-redshift-community/spark-redshift/blob/master/src/test/scala/io/github/spark_redshift_community/spark/redshift/ConversionsSuite.scala#L45

I've also tested the latest revision up to our python layer internally and timestamptz parsing works (scroll right):
```
+--------+--------+----------+-----------------+---------+-------+----------------+---------+-------------+--------------------+--------------------+
|testbyte|testbool|  testdate|       testdouble|testfloat|testint|        testlong|testshort|   teststring|       testtimestamp|     testtimestamptz|
+--------+--------+----------+-----------------+---------+-------+----------------+---------+-------------+--------------------+--------------------+
|    null|    null|      null|             null|     null|   null|            null|     null|         null|                null|                null|
|       0|    null|2015-07-03|              0.0|     -1.0|4141214|1239012341823719|     null|            f| 2015-07-03 00:00:00| 2015-07-03 03:00:00|
|       0|   false|      null|-1234152.12312498| 100000.0|   null|1239012341823719|       24|     ___|_123|                null|                null|
|       1|   false|2015-07-02|              0.0|      0.0|     42|1239012341823719|      -13|         asdf| 2015-07-02 00:00:00| 2015-07-02 04:00:00|
|       1|    true|2015-07-01| 1234152.12312498|      1.0|     42|1239012341823719|       23|Unicode's樂趣|2015-07-01 00:00:...|2015-07-01 05:00:...|
+--------+--------+----------+-----------------+---------+-------+----------------+---------+-------------+--------------------+--------------------+
```